.github/workflows/test.yml

```diff
@@ -64,6 +64,9 @@ jobs:
       - name: Run unit tests
         uses: "./.github/actions/unit_tests"
 
+      - name: Run bundlesize test
```

Comment:
Knowing there is a difference between generated-project and the template, does that mean we can't reply on the `npm run analyze-build` in the template anymore?

Reply:
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1121#issuecomment-1502423617

Reply:
Updated the vendor.js chunking rule, so there is no difference in CI now.